app/customer.py

```diff
@@ -0,0 +1,47 @@
+import math
+from app.main import load_config
+
+
+class Car:
+    def __init__(self, brand: str, fuel_consumption: float) -> None:
+        self.brand = brand
+        self.fuel_consumption = fuel_consumption
+
+    def __repr__(self) -> str:
+        return f"{self.brand}, {self.fuel_consumption}"
+
+
+class Customer:
+    def __init__(
+        self,
+        name: str,
+        product_cart: dict,
+        money: int,
+        car: dict,
+        location: list = None,
+    ) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.money = money
+        self.car: Car = Car(car["brand"], car["fuel_consumption"])
+        self.location = location if location else [0, 0]
+
+    def __repr__(self) -> str:
+        return f"{self.name}," \
+               f" {self.product_cart}," \
+               f" {self.location}," \
+               f" {self.money}," \
+               f" {self.car}"
+
+    def get_fare(self, customers: list) -> float:

```

Comment:
customer accepts customer list?

Reply:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contain string elements has typehint list[str]